pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -821,24 +821,22 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes).To(BeEmpty())
 		})
 		It("should reject bigger guest memory than the memory limit", func() {
-			vmi := api.NewMinimalVMI("testvmi")
-			guestMemory := resource.MustParse("128Mi")
-
+			vmi := newBaseVmi()
 			vmi.Spec.Domain.Resources.Limits = k8sv1.ResourceList{
 				k8sv1.ResourceMemory: resource.MustParse("64Mi"),
 			}
-			vmi.Spec.Domain.Memory = &v1.Memory{Guest: &guestMemory}
 
 			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vmi.Spec, config)
 			Expect(causes).To(HaveLen(1))
-			Expect(causes[0].Field).To(Equal("fake.domain.memory.guest"))
+			Expect(causes[0].Field).To(Equal("fake.domain.resources.requests.memory"))
```

Comment:
This change also changes the test itself. Please introduce it in a dedicated commit with a proper reasoning.

Reply:
thanks, you right!
I don't have proper reasoning actually, i mistakenly thought i could take advantage of the `libvmi.WithResourceMemory("512Mi")` that takes place in the `newBaseVmi` helper.